.eslintrc.js

```diff
@@ -29,7 +29,11 @@ module.exports = {
 
     'import/no-extraneous-dependencies': ['error', { devDependencies: [
       // only allow devDependencies in these folders:
-      '*.js', '**/__tests__/*', 'frontend/**/*', 'jest/**/*'
+      '*.js',
+      '**/__tests__/*',
+      'jest/**/*',
+      '**/*.stories.jsx',
```

Comment:
And yet,
```
image-annotator/frontend/components/ProgressBar/ProgressBar.stories.jsx
  2:1  error  '@kadira/storybook' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
```
Totally stumped here, passing this to `glob` manually finds the right file, I don't know why ESLint isn't whitelisting this pattern.